vultr/resource_block_storage.go

```diff
@@ -0,0 +1,169 @@
+package vultr
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceBlockStorage() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceBlockStorageCreate,
+		Read:   resourceBlockStorageRead,
+		Update: resourceBlockStorageUpdate,
+		Delete: resourceBlockStorageDelete,
+
+		Schema: map[string]*schema.Schema{
+			"subid": {
```

Comment:
We should remove this field, it duplicates the .id field that we set with `.SetId(...)`. Rather than access the I’d with `vultr_block_storage.example.suid` they can use the built in `vultr_block_storage.example.id`